packages/base/src/tools.ts

```diff
@@ -457,6 +460,33 @@ export const loadGeoTIFFWithCache = async (sourceInfo: {
   };
 };
 
+const getFromCache = async (key: string) => {
```

Comment:
Small nitpick, I'm a bit confused about these wrappers around `getFromIndexedDB`, shouldn't you modify `getFromIndexedDB` and `saveToIndexedDB` without introducing those wrappers?

Reply:
Ahh, you're right. I didn't realize this when taking out this code from `loadGeoTIFFWithCache`, I just tried your suggestion and removed these wrappers.